app/adapters/signal_adapter/inbound.rb

```diff
@@ -68,7 +68,12 @@ def initialize_message(signal_message)
 
       data_message = signal_message.dig(:envelope, :dataMessage)
 
-      message = Message.new(text: data_message[:message], sender: sender)
+      reaction = data_message[:reaction]
+      return nil if reaction && reaction[:isRemove]
```

Comment:
Maybe add some consistency here? Either `return` like :point_up: or change the lines above to `return nil` as well.

Reply:
Not a hundred percent sure what you exactly you mean (which lines should return `nil`? because there isn't really another place where we should return `nil` or?)

But I assume this is mostly about the clarity / consistency to know when nil is returned and that can definitely be improved - I pushed a new version which moves all checks for when it's appropriate to return nil to the top of the method. What do you think ?